SwiftSyntaxDevUtils/Sources/swift-syntax-dev-utils/commands/Test.swift

```diff
@@ -0,0 +1,172 @@
+        message: """
+          Error: Could not find incr_transfer_round_trip.py.
+
+          Make sure you have the main swift repo checked out next to the swift-syntax
+          repo.
+          Refer to README.md for more information.
+          """
+      )
+    }
+
+    let productBinPath = try findProductBinPath()
+    let examplesBinPath = try findExamplesBinPath()
+
+    let litTestHelperExec = productBinPath.appendingPathComponent("/lit-test-helper")
+    var litCall = [
+      Paths.litExec.path,
+      Paths.packageDir.path,
+      "lit_tests",
+    ]
+
+    if let filecheckExec {
+      litCall += ["--param", "FILECHECK=" + filecheckExec]
+    }
+
+    litCall += ["--param", "LIT_TEST_HELPER=" + litTestHelperExec.path]
+    litCall += ["--param", "INCR_TRANSFER_ROUND_TRIP.PY=" + Paths.incrTransferRoundtripExec.path]
+
+    litCall += ["--param", "EXAMPLES_BIN_PATH=" + examplesBinPath.path]
+    litCall += ["--param", "TOOLCHAIN=" + arguments.toolchain.path]
+
+    // Print all failures
+    litCall += ["--verbose"]
+    // Don't show all commands if verbose is not enabled
+    if !arguments.verbose {
+      litCall += ["--succinct"]
+    }
+
+    guard let pythonExec = Paths.pythonExec else {
+      throw ScriptExectutionError(message: "Didn't find python executable")
+    }
+
+    let process = ProcessRunner(
+      executableURL: pythonExec,
+      arguments: litCall
+    )
+
+    try process.run()
+  }
+
+  private func runXctests() throws {
+    logSection("Running XCTests")
+    var swiftpmCallArguments: [String] = []
+
+    if arguments.verbose {
+      swiftpmCallArguments += ["--verbose"]
+    }
+
+    swiftpmCallArguments += ["--test-product", "SwiftSyntaxPackageTests"]
+
+    var additionalEnvironment: [String: String] = [:]
+    additionalEnvironment["SWIFT_BUILD_SCRIPT_ENVIRONMENT"] = "1"
+
+    if arguments.enableRawsyntaxValidation {
+      additionalEnvironment["SWIFTSYNTAX_ENABLE_RAWSYNTAX_VALIDATION"] = "1"
+    }
+
+    if arguments.enableTestFuzzing {
+      additionalEnvironment["SWIFTPARSER_ENABLE_ALTERNATE_TOKEN_INTROSPECTION"] = "1"
+    }
+
+    // Tell other projects in the unified build to use local dependencies
+    additionalEnvironment["SWIFTCI_USE_LOCAL_DEPS"] = "1"
+    additionalEnvironment["SWIFT_SYNTAX_PARSER_LIB_SEARCH_PATH"] =
+      arguments.toolchain
+      .appendingPathComponent("lib")
+      .appendingPathComponent("swift")
+      .appendingPathComponent("macosx")
+      .path
+
+    try invokeSwiftPM(
+      action: "test",
+      packageDir: Paths.packageDir,
+      additionalArguments: swiftpmCallArguments,
+      additionalEnvironment: additionalEnvironment
+    )
+  }
+
+  private func findSwiftpmBinPath(packageDir: URL) throws -> String {
+    return try invokeSwiftPM(
+      action: "build",
+      packageDir: packageDir,
+      additionalArguments: ["--show-bin-path"],
+      additionalEnvironment: [:]
+    )
+  }
+
+  private func findProductBinPath() throws -> URL {
```

Comment:
Could you add a short comment which path this points to, ie. an example path and/or what the path contains.